src/components/forms/pool_actions/WithdrawForm/components/WithdrawTotals.vue

```diff
@@ -28,6 +28,7 @@ const priceImpactClasses = computed(() => ({
       <div class="data-table-number-col">
         <div class="flex items-center">
           <BalLoadingBlock v-if="isLoadingQuery" class="w-10 h-6" />
+          <span v-else-if="priceImpact === -1">-</span>
```

Comment:
I would extract the -1 to a semantic constant (i.e. `undefinedPriceImpact`) to make this logic clearer.

Reply:
Yeah fair enough. I was being lazy... 😄